java-hello-world-maven/pom.xml

```diff
@@ -17,7 +17,7 @@
     <maven.compiler.source>17</maven.compiler.source>
     <maven.compiler.target>17</maven.compiler.target>
     <maven-jar-plugin.version>3.2.2</maven-jar-plugin.version>
-		<native.maven.plugin.version>0.9.13</native.maven.plugin.version>
+		<native.maven.plugin.version>0.9.28</native.maven.plugin.version>
```

Comment:
@LesiaChaban, I checked the GitHub Action build output, and it printed this message: 
"Warning:  'native:build' goal is deprecated. Use 'native:compile-no-fork' instead."

Reply:
Update pom.xml replacing `build` with `compile-no-fork` at line 66.